massgen/tests/test_file_overwrite_protection.py

```
@@ -0,0 +1,240 @@
+
+        assert allowed is True
+        assert reason is None
+
+    def test_edit_tool_claude_code_allowed(self):
+        """Test that Claude Code 'Edit' on existing file is allowed."""
+        manager = self.helper.create_permission_manager()
+        existing_file = self.helper.workspace_dir / "code.py"
+        existing_file.write_text("# code")
+
+        tool_args = {"file_path": str(existing_file)}
+        allowed, reason = manager._validate_write_tool("Edit", tool_args)
```

Ruff (0.14.10)

Unpacked variable `reason` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)